From ccc4099c71ba0cb154b4cfdc4608190eda5b75fd Mon Sep 17 00:00:00 2001
From: Scott K Logan <logans@cottsay.net>
Date: Fri, 22 Mar 2024 20:42:47 -0500
Subject: [PATCH] Install the package.xml manifest

---
 rpm/ros-rolling-cyclonedds.spec | 1 +
 1 file changed, 1 insertion(+)

diff --git a/rpm/ros-rolling-cyclonedds.spec b/rpm/ros-rolling-cyclonedds.spec
index e574c716e3c6..1fccfa8eef4c 100644
--- a/rpm/ros-rolling-cyclonedds.spec
+++ b/rpm/ros-rolling-cyclonedds.spec
@@ -72,6 +72,7 @@ IoT project.
 
 %install
 %cmake_install
+install -m0644 -p -D package.xml %{buildroot}%{bloom_prefix}/share/%{pkg_name}/package.xml
 
 
 %if 0%{?with_tests}
-- 
2.44.0

